xGame/client/src/Components/Margins/Header.js

```diff
@@ -123,6 +136,11 @@ const Header = (props) => {
                       </Button>
                     </CardActions>
                   )}
+                  {!isInvitation && (
+                    <IconButton onClick={() => readOneNotification(notification?.id)}>
```

Comment:
Can you put this on the same line as the "Message" text? It's a little confusing being below all the text